### PR TITLE
Search Panel: Advanced Search Not Displayed

### PR DESCRIPTION
**Pull request by flavens**
closes [DSP-590](https://dasch.myjetbrains.com/youtrack/issue/DSP-590)

### CODE REVIEW

File: projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html [hunk 1]
```diff
@@ -44,9 +44,8 @@ <h4 *ngIf="!showAdvanced">Expert search</h4>
             </span>
         </div>
         <div class="dsp-menu-content">
-            <!-- TODO: reactivate this tags when the components are ready -->
-            <!-- <dsp-advanced-search *ngIf="showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-advanced-search> -->
-            <!-- <dsp-expert-search *ngIf="!showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-expert-search> -->
+            <dsp-advanced-search *ngIf="showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-advanced-search>
```
Comment (tobiasschweizer) [inline projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html:47]: Could you put `search.emit($event); closeMenu()` in a method? So then you could call in both cases: advanced and expert search.

File: projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html [hunk 2]
```diff
@@ -44,9 +44,8 @@ <h4 *ngIf="!showAdvanced">Expert search</h4>
...
+            <dsp-expert-search *ngIf="!showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-expert-search>
```
Comment (tobiasschweizer) [inline projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html:48]: Shouldn't this check for the input `expert`?

**Review by tobiasschweizer — CHANGES_REQUESTED** [review]

File: projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html (hunk 1 above)
Comment (flavens) [reply projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html]: fixed in 139009f

File: projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html (hunk 2 above)
Comment (flavens) [reply projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html]: I do not understand what you mean but the flag showAdvanced is doing the check (advanced or expert search).

Comment (tobiasschweizer) [reply projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html]: But couldn't you use [`*ngIf` with an else block](https://angular.io/api/common/NgIf#description) to make this clearer?

Comment (tobiasschweizer) [reply projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html]: I was just wondering whether the `*ngIf ... else` syntax would be clearer than two `*ngIf`s.

Comment (flavens) [reply projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html]: I would keep it this way

**Review by tobiasschweizer — APPROVED** [review]
ok, works for me in BEOL now.

**Pull request merged**